README.md

```diff
@@ -1,12 +1,18 @@
 The 18F Method Cards (“18F Methods”) describe how 18F puts human-centered design into practice. While this is primarily maintained as an internal resource, we hope it can help *everyone* adopt the methods of human-centered design.
 
-### Why methods?
+## 18F Methods mission and vision
+In late 2016, we articulated a vision and mission for the project that focus our efforts on meeting the needs of government users:
 
-In order to function well within cross-functional teams, designers need to know a few things: which methods they might choose from, why one particular method makes more sense than another at any given moment, and, once they’ve picked a method, how to actually execute it. 18F Methods collects this essential information as a series of cards. In practice, we’ve found the Methods can provide folks with a gateway into our work and build internal alignment around a shared vocabulary.
+ - **Vision:** To help folks choose and apply human centered design approaches for solving problems in government.
+ - **Mission:** To increase adoption of human centered design tools by providing guidance on sequencing and adapting these tools within the constraints of a given government project.
```

Comment:
Suggest changing mission to read `To increase adoption and offer guidance on applying human-centered design to government projects.`

Reply:
This one doesn't feel as pithy to me.
Define increased adoption? (of...)
and perhaps "... guidance on how to apply human....)

Reply:
Any thoughts on the above @andrewmaier and @ericronne do you have a suggestion on changes?

Reply:
This is a 🐻 of a task, innit!
Another crack, to keep the convo rolling …

`To offer civil servants simple and practical techniques for their design projects — categorized and sequenced to aid in selection — along with guidance to help them use within the constraints of government.`

Reply:
Riffing off of what you have there @ericronne 

`
To offer simple and practical techniques for selecting and implementing human-centered design methods within the constraints of government.
`

Thoughts?
CC @andrewmaier

Reply:
brills! 💥